crates/turborepo-scm/src/git.rs

```diff
@@ -64,50 +68,53 @@ pub fn changed_files(
 fn get_stripped_system_file_path(
     repo_root: &ProjectRoot,
     file_path: &Path,
-    monorepo_root: &ProjectRelativePath,
+    turborepo_root: &AbsoluteNormalizedPath,
 ) -> Result<PathBuf, Error> {
     // We know the path is relative to the repo root so we can convert it to a
     // ProjectRelativePath
     let project_relative_file_path = ProjectRelativePath::new(file_path)?;
     // Which we then resolve to an absolute path
     let absolute_file_path = repo_root.resolve(project_relative_file_path);
```

Comment:
Reading through more of `turborepo_path`, I'm surprised that we need to use `Path::from_slash`: https://github.com/vercel/turbo/blob/main/crates/turborepo-paths/src/absolute_normalized_path.rs#L131